client/src/components/Meal.jsx

```diff
@@ -94,13 +152,104 @@ export function Meal({ meal }) {
         </CalorieText>
       </MealDetail>
       <ButtonWrapper>
-        <ActionButton>
-          <DoneIcon />{' '}
+        <ActionButton onClick={onEditOpen}>
+          <EditButton />{' '}
         </ActionButton>
-        <ActionButton>
+        <ActionButton onClick={() => onDeleteMeal(meal.id)}>
```

Comment:
handleDeleteMeal

თუ სხვა ქოლბექებს ცალკე აღწერ, ამან რა დაგიშავა. ასეთი მექანიკური inconsistent შეცდომები მიანიშნებს იმაზე, რომ ზერელედ წერ კოდს. კარგი თიმი ამაზე ყურადღებას გაგიმახვილებს აუცილებლად

Reply:
ინლაინ რომ არის მაგიტომ არის inconsistent ?
ანუ ცალკე აღვწერო ეგ ჰენდლერი და იქ  onDeleteMeal ქოლბექი
### is this correct?
![Screenshot 2022-02-06 192216](https://user-images.githubusercontent.com/68153140/152687895-bcbd5b43-6e75-4294-9c2c-13ebfaac70ef.png)
olution?